game.py

```diff
@@ -102,6 +103,93 @@ def _action_to_steps_move(self, action: Action) -> list[int]:
     def _steps_move_to_action(self, steps_move: list[int]) -> Action:
         """Convert a pydraughts move to something usable by the model"""
         return (steps_move[0] - 1) * self.BOARD_SIZE + steps_move[1] - 1
+    
+    def get_count(self, Player) -> int:
+        """
+        Counts the total number of pieces for WHITE or BLACK
+        """
+        if Player == Player.WHITE :
+            player = 2
+        else:
+            player = 1
```

Comment:
you can replace the magic numbers 2 and 1 with values `draughts.WHITE` and `draugts.BLACK`, which is a bit easier to read